templates/cfp/list.twig

```diff
@@ -26,22 +26,19 @@
  * @since     18.01.2016
  * @link      http://github.com/heiglandreas/callingallpapers
  *#}
-{% macro recursiveItem(arry) %}
-{% for key, value in arry %}
+{% for pairs in cfps %}
+<h1>{{ pairs.name }}</h1>
 <dl>
-  <dt>
-  {{ key }}:
-  </dt>
-  <dd>
-  {% if value is iterable %}
-    {{ _self.recursiveItem(value) }}
-  {% else %}
-    {{ value }}
+  {% for key, value in pairs %}
+    <dt>
+      {{ key }}:
+    </dt>
+    <dd>
+      {{ value|join(', ', ' and ') }}
+    </dd>
+  {% endfor %}
+  {% if not loop.last %}
+    <hr>
```

Comment:
Can we remove that `<hr>` from the definition-list and instead add some CSS? Like a border-top except for the first-child?

Reply:
Fixed!